backend/nlp/handler.py

```diff
@@ -99,13 +112,32 @@ def analyze_reviews(reviews_list):
                 #if abs(sentiment_scores['compound']) > abs(compound):
                 compound += sentiment_scores['compound']
 
-            processed_review['neg'] = bad
-            processed_review['pos'] = good
-            processed_review['compound'] = compound
+            review['negSentiment'] = int(bad * 100)
+            review['posSentiment'] = int(good * 100)
+            review['compSentiment'] = int(compound * 100)
+
 
-            list_processed_reviews.append(processed_review)
+            if review["review"]["title"].strip() == "":
+                review["review"]["title"] = "No Title Given"
```

Comment:
Is there a better value we could use when there's no title? I know DynamoDB doesn't support empty strings, but maybe we can use `null`?

Reply:
ok that works, I was just getting the data to pass through, null is a good idea